moonstone/parsers/counts/taxonomy/metaphlan3.py

```diff
@@ -13,11 +13,52 @@ class Metaphlan3Parser(TaxonomyCountsBase, BaseParser):
     NCBI_tax_column = 'NCBI_tax_id'
 
     def __init__(self, *args, **kwargs):
+        self.analysis_type = kwargs['analysis_type']
+        kwargs.pop('analysis_type')
         super().__init__(*args, parsing_options={'skiprows': 1}, **kwargs)
 
+    def remove_duplicates(self, df):
```

Comment:
What you could also do is test this method only individually in your unit tests to give more information about what is going on with this method.